src/main/java/com/example/TicketFinder/Concert.java

```diff
@@ -0,0 +1,19 @@
+package com.example.TicketFinder;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+class Concert {
+    private String nameOfConcert;
```

Comment:
you don't need to add a suffix when the field is a part of the class. You should rename it to just `name`